### DISCUSSION TITLE
Chia plotting stops when starting more than 1 plot in parallel

### DISCUSSION THREAD

**Discussion by YDDY1** [Chia-Network/chia-blockchain#3884] [Support]
hello how recently when I try to start more than 1 plots in parallel they slow down and it takes more than 24 hours to even advance 1 percent, and it went from having a good use of processor to having less than 3 percent it also happens to use less than 1gb of ram. I have a 2tb nvme ssd, 64gb of ram and an 8-core processor, and I tried with different configurations to make the plots and it always seems to occur the same error which did not happen previously.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
